stock_cycle_count/models/stock_location.py

```diff
@@ -115,9 +115,9 @@ def action_accuracy_stats(self):
         self.ensure_one()
         action = self.env.ref("stock_cycle_count.act_accuracy_stats")
         result = action.read()[0]
-        result["context"] = {"search_default_location_id": self.id}
+        result["context"] = {"search_default_location_ids": self.name}
         new_domain = (
-            result["domain"][:-1] + ", ('location_id', 'child_of', active_ids)]"
+            result["domain"][:-1] + ", ('location_ids', 'child_of', active_ids)]"
```

Comment:
Not sure if operator `child_of` works with many2many fields.

Reply:
I tested changes before create PR and working fine
See 
https://github.com/odoo/odoo/blob/13.0/odoo/osv/expression.py#L1009